### DISCUSSION TITLE
Hard to read the documentation.

### DISCUSSION THREAD

**Discussion by No-518** [synsense/sinabs#220] [Q&A]
The Document of Sinabs is hard to read: https://sinabs.readthedocs.io/en/v1.2.9/. This page is written in a very difficult way, can you change its layout?

**Comment by bauerfe** [comment]
Thank you very much for your feedback! 
In order to improve the documentation, it would be very helpful for us to know where exactly the difficulty lies. 
Could you please elaborate a bit more on that?

**Comment by sheiksadique** [comment]
Hi @No-518, Thankyou for this feedback. We always aspire to provide the best documentation possible to our users. Could you give us a bit more information about what specifically you found difficult and if you have a recommendation for how you would to see it changed?
You are also welcome to create a pull request and contribute to the improvement to the documentation! This would be highly appreciated and useful for us and the community.
Thankyou!

**Comment by bauerfe** [comment]
Since we haven't heard back on this issue, I will close it. However, @No-518 , feel free to re-open or open a new issue anytime you would like to share further feedback or to contribute.
